frontend/fingertips-frontend/components/organisms/BenchmarkLabel/BenchmarkLabelConfig.ts

```diff
@@ -3,19 +3,29 @@ import {
   QuintileColours,
   TagColours,
 } from '@/lib/styleHelpers/colours';
+import { BenchmarkLabelGroupConfig } from '@/components/organisms/BenchmarkLabel/BenchmarkLabelTypes';
 import {
-  BenchmarkLabelGroupConfig,
-  BenchmarkLabelGroupType,
-  BenchmarkLabelType,
-} from '@/components/organisms/BenchmarkLabel/BenchmarkLabelTypes';
+  BenchmarkComparisonMethod,
+  BenchmarkOutcome,
+  IndicatorPolarity,
+} from '@/generated-sources/ft-api-client';
 
 export const getBenchmarkTagStyle = (
-  group: BenchmarkLabelGroupType,
-  type: BenchmarkLabelType
+  group: BenchmarkComparisonMethod,
+  type: BenchmarkOutcome,
+  polarity: IndicatorPolarity
 ) => {
   const groupConfig = benchmarkLabelGroupConfig[group];
   if (!groupConfig) return null;
 
+  // special case Middle is used by Quintiles with and without judgement but has different colours
+  if (
+    polarity !== IndicatorPolarity.NoJudgement &&
+    type === BenchmarkOutcome.Middle
+  ) {
+    return groupConfig['middleWithJudgement'];
+  }
+
   const typeConf = groupConfig[type];
   return typeConf ?? groupConfig.default;
```

Comment:
Would you mind tidying this up to 
```suggestion
  return groupConfig[type] ?? groupConfig.default;
```